autogen/oai/gemini.py

```diff
@@ -209,99 +250,111 @@ def create(self, params: Dict) -> ChatCompletion:
 
         return response_oai
 
+    def oai_content_to_gemini_content(self, content: Union[str, List]) -> List:
```

Comment:
Let's rename this function to "_oai_content_to_gemini_content"

Reply:
The function is now private